Source/Scene/Cesium3DTileset.js

```diff
@@ -102,6 +102,7 @@ import TileOrientedBoundingBox from "./TileOrientedBoundingBox.js";
  * @param {Boolean} [options.debugShowRenderingStatistics=false] For debugging only. When true, draws labels to indicate the number of commands, points, triangles and features for each tile.
  * @param {Boolean} [options.debugShowMemoryUsage=false] For debugging only. When true, draws labels to indicate the texture and geometry memory in megabytes used by each tile.
  * @param {Boolean} [options.debugShowUrl=false] For debugging only. When true, draws labels to indicate the url of each tile.
+ * @param {Boolean} [options.noClassificationModels=false] Do not use b3dms for classification.
```

Comment:
Looking at the commit history `noClassificationModels` is meant for https://github.com/CesiumGS/cesium/pull/9398 and it's a way to tell a tileset that has both b3dms and vctr to not treat the b3dms as classification models. Is there a less hacky way to do this?

CC @likangning93

Reply:
Maybe #9398 should have a separate `classificationType` for vector polylines?